src/benchcab/data/config-schema.yml

```diff
@@ -107,6 +107,9 @@ fluxsite:
           schema:
             type: "string"
             required: false
+    meorg_model_output_id:
```

Comment:
New schema keys, optional `model_output_id`

Reply:
Please document the behaviour of `model_output_id` in the docs (i.e. https://benchcab.readthedocs.io/en/latest/user_guide/config_options/)